sys/tiny_strerror/tiny_strerror.c

```diff
@@ -117,6 +118,12 @@ const char *tiny_strerror(int errnum)
         errnum = -errnum;
     }
 
+    if (IS_USED(MODULE_TINY_STRERROR_MINIMAL)) {
+        static char buf[4];
+        snprintf(buf, sizeof(buf), "%u", (unsigned)errnum);
```

Comment:
```suggestion
        snprintf(buf, sizeof(buf), "%d", errnum);
```

How about keeping the sign (and moving it up before the `if (errnum <= 0) {`)? That would be consistent with the regular version and would not cost ROM size.

The `vfs` app previously provided the minus char for negative values, which is why `tiny_strerror()` also provides it. But I think this is actually a nice feature.